src/widgets/appointments/appointment-record.component.tsx

```diff
@@ -127,18 +127,18 @@ export default function AppointmentRecord(props: AppointmentRecordProps) {
               <tbody>
                 <tr>
                   <td>
-                    {dayjs(patientAppointment.startDateTime).format(
+                    {dayjs(patientAppointment?.startDateTime).format(
                       "DD-MMM-YYYY"
                     )}
                   </td>
                   <td>
                     {patientAppointment.service.creatorName
```

Comment:
I guess we miss the conditional member operator here `?.`. Actually I think the three lines could be replaced by

```ts
patientAppointment?.service?.creatorName || "\u2014"
```

Reply:
Yeah. Elegant solution.